Package.swift

```diff
@@ -4,12 +4,6 @@ import PackageDescription
 
 let package = Package(
     name: "MUXSDKStats",
-    platforms: [
-        .iOS(.v12),
-        .macCatalyst(.v13),
-        .tvOS(.v12),
-        .visionOS(.v1),
-    ],
```

Comment:
These are the default minimum versions, so this block does nothing. It's considered best practice for a utility package like ours to omit these and use availability annotations on individual APIs.